### PR TITLE
MINOR: apply Utils.isBlank to code base

### PR DESCRIPTION
**Pull request by tang7526**
apply Utils.isBlank to code base

### Committer Checklist (excluded from commit message)
- [ ] Verify design and implementation 
- [ ] Verify test coverage and CI build status
- [ ] Verify documentation (including upgrade notes)

### CODE REVIEW

File: clients/src/main/java/org/apache/kafka/clients/consumer/KafkaConsumer.java [hunk 1]
```diff
@@ -1106,11 +1107,13 @@ public void assign(Collection<TopicPartition> partitions) {
             } else if (partitions.isEmpty()) {
                 this.unsubscribe();
             } else {
-                for (TopicPartition tp : partitions) {
+                partitions.forEach(tp -> {
```
Comment (chia7712) [inline clients/src/main/java/org/apache/kafka/clients/consumer/KafkaConsumer.java]: It would be nice to make related changes only :)

Comment (tang7526) [reply clients/src/main/java/org/apache/kafka/clients/consumer/KafkaConsumer.java]: OK. I have reverted un-related changes.

**Comment by chia7712** [comment]
@tang7526 Thanks for your patch. Could you fix following code also?

1. https://github.com/apache/kafka/blob/trunk/connect/runtime/src/main/java/org/apache/kafka/connect/runtime/isolation/Plugins.java#L448
1. https://github.com/apache/kafka/blob/trunk/clients/src/test/java/org/apache/kafka/common/security/oauthbearer/internals/unsecured/OAuthBearerScopeUtilsTest.java#L31
1. https://github.com/apache/kafka/blob/trunk/connect/api/src/test/java/org/apache/kafka/connect/data/ValuesTest.java#L914
1. https://github.com/apache/kafka/blob/trunk/core/src/main/scala/kafka/network/SocketServer.scala#L642
1. https://github.com/apache/kafka/blob/trunk/core/src/main/scala/kafka/server/KafkaServer.scala#L473
1. https://github.com/apache/kafka/blob/trunk/core/src/main/scala/kafka/utils/Log4jController.scala#L74
1. https://github.com/apache/kafka/blob/trunk/core/src/main/scala/kafka/utils/Log4jController.scala#L83

**Comment by tang7526** [comment]
> @tang7526 Thanks for your patch. Could you fix following code also?
> 
> 1. https://github.com/apache/kafka/blob/trunk/connect/runtime/src/main/java/org/apache/kafka/connect/runtime/isolation/Plugins.java#L448
> 2. https://github.com/apache/kafka/blob/trunk/clients/src/test/java/org/apache/kafka/common/security/oauthbearer/internals/unsecured/OAuthBearerScopeUtilsTest.java#L31
> 3. https://github.com/apache/kafka/blob/trunk/connect/api/src/test/java/org/apache/kafka/connect/data/ValuesTest.java#L914
> 4. https://github.com/apache/kafka/blob/trunk/core/src/main/scala/kafka/network/SocketServer.scala#L642
> 5. https://github.com/apache/kafka/blob/trunk/core/src/main/scala/kafka/server/KafkaServer.scala#L473
> 6. https://github.com/apache/kafka/blob/trunk/core/src/main/scala/kafka/utils/Log4jController.scala#L74
> 7. https://github.com/apache/kafka/blob/trunk/core/src/main/scala/kafka/utils/Log4jController.scala#L83

@chia7712 Done

**Review by chia7712 — COMMENTED** [review]
@tang7526 thanks for your patch. LGTM

**Review by chia7712 — APPROVED** [review]

**Comment by chia7712** [comment]
`kafka.admin.ListConsumerGroupTest.testListConsumerGroupsWithStates()` unrelated error

**Pull request merged**